enhancements/api-review/add-new-CRD-ImageSourceDigestPolicy-and-ImageSourceTagPolicy-to-config.openshift.io.md

```diff
@@ -0,0 +1,393 @@
+---
+title: add-new-CRD-ImageSourceDigestPolicy-and-ImageSourceTagPolicy-to-config.openshift.io
+authors:
+  - "@QiWang19"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+approvers:
+  - TBD
+  - "@oscardoe"
+creation-date: 2021-03-10
+last-updated: 2021-11-11
+status: implementable
+---
+
+# Add CRD ImageSourceDigestPolicy and ImageSourceTagPolicy to config.openshift.io/v1
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Today, the ImageContentSourcePolicy(ICSP) object sets up mirror with `mirror-by-digest-only` property set to true. Requires images be pulled by digest only. This enhancement introduce new CRDs to config/v1:
```

Comment:
```suggestion
Today, the ImageContentSourcePolicy (ICSP) object sets up mirror with `mirror-by-digest-only` property set to true. Requires images be pulled by digest only. This enhancement introduce new CRDs to config/v1:
```

Reply:
Since you are declaring the acronym ICSP here, you can just use it throughout the enhancement instead of the full name.